int16 auto_park_status

